<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QWidget" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>200</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Overpayment</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_total_sum">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_overpayment">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Total sum</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Mothly payment</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_month_pay">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_amount">
       <property name="text">
        <string>100000</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_rate">
       <property name="text">
        <string>12</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rate, %</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_type">
       <item>
        <property name="text">
         <string>Annuity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Differentaited</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Timeframe</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_timeframe">
       <property name="currentIndex">
        <number>6</number>
       </property>
       <item>
        <property name="text">
         <string>1 month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6 month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9 month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30 year</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_calculate">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
